src/config.rs

```diff
@@ -354,6 +354,8 @@ pub struct Misc {
 pub enum TmuxSessionMode {
     AttachIfNotInSession,
     AttachAlways,
+    ReattachIfNotInSession,
+    ReattachAlways,
```

Comment:
Instead of expanding this enum, could you make a new `tmux_reattach: bool` (or a different name if you like)? This would also remove the duplication that is currently in the `match` block.